tools/mo/openvino/tools/mo/utils/ir_engine/compare_graphs.py

```diff
@@ -116,6 +116,24 @@ def compare_graphs(graph: Graph, graph_ref: Graph, last_node: str, last_node_ref
                                       ''.format(node.id, cur_node_type, attr))
                         continue
 
+                    def align_strided_slice_masks(curr_node: Node, rank: int):
```

Comment:
Nice to have comment about what this alignment means and why we do it for strided slice

Reply:
Done.